packages/paste-website/src/pages/layout/grid/index.mdx

```diff
@@ -164,6 +164,49 @@ Use `offset` to push a column to the right by a certain number of columns. For e
 </Grid>`}
 </LivePreview>
 
+### Equal column heights
+
+Set `equalColumnHeights` on a grid container to set the height all columns in a single row to be equal to the height of the tallest column.
```

Comment:
Do we need to add a note on setting size on child elements?

Reply:
What about now?